FrontEnd/src/components/Contact/Contact.jsx

```diff
@@ -0,0 +1,31 @@
+import styles from './Contact.module.css';
+import contactText from './text';
+import Text from './text';
+import { v4 as uuidv4 } from 'uuid';
+import { useEffect } from 'react';
+import ContentRenderer from '../CookiesPolicyPage/RenderingTextContainer.jsx';
+import LinkContainer from '../CookiesPolicyPage/LinkContainer.jsx';
+
+const Contact = () => {
+    useEffect(() => {
+        window.scrollTo(0, 0);
+    }, []);
+  return (
```

Comment:
nit: It seems not to be aligned. Could you format it?

Reply:
done